From: Shengjing Zhu <zhsj@debian.org>
Date: Wed, 23 Aug 2023 14:07:38 +0800
Subject: sys: unprivileged user can adjust oom scroe when initial is negative

Forwarded: https://github.com/containerd/containerd/pull/8999
---
 sys/oom_linux_test.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sys/oom_linux_test.go b/sys/oom_linux_test.go
index 0c8f90b..170aa98 100644
--- a/sys/oom_linux_test.go
+++ b/sys/oom_linux_test.go
@@ -55,6 +55,9 @@ func TestSetNegativeOomScoreAdjustmentWhenUnprivilegedHasNoEffect(t *testing.T)
 
 	initial, adjustment, err := adjustOom(-123)
 	assert.NilError(t, err)
+	if initial < 0 {
+		t.Skip("unprivileged user can adjust oom scroe when initial is negative")
+	}
 	assert.Check(t, is.Equal(adjustment, initial))
 }
 
